popup.js

```diff
@@ -36,17 +37,36 @@ function getExp(skillName, callback){
   });
 }
 
+function getLevel(exp) {
+  /* Calculates current level and exp needed to next level.
+   * O(max_level) - computational complexity
+   */
+  var level = 0;
+  for (var i = 0; i < 200; i++){
```

Comment:
Why this value is 200?

Reply:
If it is really necessay to use this value add global const for program.
[Edit] As I understood this is max possible level. If you should at this moment the best solution would be global const `MAXLEVEL`. You can read more about it here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const